### PR TITLE
Homework9: Alexey Kozhan

### PR DESCRIPTION
**Pull request by AlexKozhan**
homework9 is done in 3 Python files

### CODE REVIEW

File: homework1.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    for i in a:
+        if i == '#':
+            if masiv:
+                masiv.pop()
+        else:
+            masiv.append(i)
+    return ''.join(masiv)
+
+
+print(sortirovka("a#bc#d"))
```
Comment (Stml89) [inline homework1.py:29]: use `assert` instead of `print()`

File: homework1.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+"a#bc#d" ==>  "bd"
+"abc#d##c"      ==>  "ac"
+"abc##d######"  ==>  ""
+"#######"       ==>  ""
+""              ==>  ""
+"""
+
+
+def sortirovka(a):
```
Comment (Stml89) [inline homework1.py:16]: a better name is `sorting`, `remove_hashes` and so on

File: homework2.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+assert solution(13, 5) == 16
+assert solution(2, 3) == 2
+
+print(solution(5, 2))
+print(solution(1, 2))
+print(solution(15, 5))
+print(solution(12, 2))
+print(solution(6, 4))
+print(solution(13, 5))
+print(solution(2, 3))
```
Comment (Stml89) [inline homework2.py:61]: remove this prints lines: 55-61

**Review by Stml89 — COMMENTED** [review]
fix comment and you can merge it

File: homework2.py (hunk 3 above)
Comment (AlexKozhan) [reply homework2.py]: fixed

File: homework1.py (hunk 2 above)
Comment (AlexKozhan) [reply homework1.py]: fixed

File: homework1.py (hunk 1 above)
Comment (AlexKozhan) [reply homework1.py]: fixed

**Pull request merged**